sig-network/charter.md

```diff
@@ -71,4 +72,6 @@ SIG Technical Leads
 
 [sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
 [sig-subprojects]: https://github.com/kubernetes/community/blob/master/sig-network/README.md#subprojects
+[sig-multicluster]:https://github.com/kubernetes/community/blob/master/sig-multicluster/README.md
```

Comment:
I think the syntax requires a space after the colon. Or at least, that's the normal style

Reply:
```suggestion
[sig-multicluster]: https://github.com/kubernetes/community/blob/master/sig-multicluster/README.md
```

Reply:
It does work without the space, however I did update it all over now to be consistent _with_ the space added :+1: